pkg/operator/highcpuusagesnoalertcontroller/performanceprofile.go

```diff
@@ -0,0 +1,61 @@
+package highcpuusagesnoalertcontroller
+
+import (
+	"context"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/utils/cpuset"
+)
+
+// default and taken from the docs
```

Comment:
why not to move the content of this file to `highcpuusagesnoalert_controller` ?
do we need a separate file for just one function ?